eng/common/scripts/Verify-Links.ps1

```diff
@@ -6,18 +6,20 @@ param (
   # switch that will enable devops specific logging for warnings
   [switch] $devOpsLogging = $false,
   # check the links recurisvely based on recursivePattern
-  [switch] $recursive = $true,
+  [switch] $recursive,
   # recusiving check links for all links verified that begin with this baseUrl, defaults to the folder the url is contained in
   [string] $baseUrl = "",
   # path to the root of the site for resolving rooted relative links, defaults to host root for http and file directory for local files
   [string] $rootUrl = "",
   # list of http status codes count as broken links. Defaults to 400, 401, 404, SocketError.HostNotFound = 11001, SocketError.NoData = 11004
   [array] $errorStatusCodes = @(400, 401, 404, 11001, 11004),
-  # flag to allow resolving relative paths or not
-  [bool] $resolveRelativeLinks = $true
+  # flag to allow checking against azure sdk link guidance.
+  [bool] $linkGuidance = $true
```

Comment:
I'd rename this to "checkLinkGuidance" and set it to false by default.

Reply:
Relative link is enabled right now. Will rename this, to make the feature backward compatible, will enable relative link and disable the check on locale